controlnet3d.py

```diff
@@ -321,7 +321,7 @@ def from_unet(
         unet: UNet3DConditionModel,
         controlnet_conditioning_channel_order: str = "rgb",
         conditioning_embedding_out_channels: Optional[Tuple[int]] = (16, 32, 96, 256),
-        load_weights_from_unet: bool = True,
+        load_weights_from_unet: bool = False,
```

Comment:
Just curious why you changed this default

Reply:
Otherwise it tries to load parts of the unet state_dict into the controlnet (not sure why, is this how controlnets are usually initialized?).

That leads to problems with the PseudoConv3D layers which have different parameter names.

We should probably figure out what exactly the difference is. Maybe that'll give us a better idea for replacing the PseudoConv3D layers with standard Diffusers versions.